at-loader errors for d.ts files

Hi,
typescript loader gives me these errors when I use aurelia-form with the typescript-webpack navigation skeleton. The aurelia app works fine.

> webpack: Compiled successfully.
> [at-loader] ./node_modules/aurelia-form/dist/aurelia-form.d.ts:216:1
>     TS1128: Declaration or statement expected.
> 
> [at-loader] ./node_modules/aurelia-form/dist/aurelia-form.d.ts:216:16
>     TS1005: ';' expected.
> 
> [at-loader] ./node_modules/aurelia-form/dist/aurelia-form.d.ts:218:8
>     TS1005: ';' expected.
> 
> [at-loader] ./node_modules/aurelia-form/dist/aurelia-form.d.ts:219:1
>     TS1128: Declaration or statement expected.
> 
> [at-loader] ./node_modules/aurelia-form/dist/aurelia-form.d.ts:219:16
>     TS1005: ';' expected.
> 
> [at-loader] ./node_modules/aurelia-form/dist/aurelia-form.d.ts:221:8
>     TS1005: ';' expected.
> 
> [at-loader] ./node_modules/aurelia-form/dist/aurelia-form.d.ts:222:1
>     TS1128: Declaration or statement expected.
> 
> [at-loader] ./node_modules/aurelia-form/dist/aurelia-form.d.ts:222:16
>     TS1005: ';' expected.
> 
> [at-loader] ./node_modules/aurelia-form/dist/aurelia-form.d.ts:224:8
>     TS1005: ';' expected.
> 
> [at-loader] ./node_modules/aurelia-form/dist/aurelia-form.d.ts:225:1
>     TS1128: Declaration or statement expected.
> 
> [at-loader] ./node_modules/aurelia-form/dist/aurelia-form.d.ts:225:16
>     TS1005: ';' expected.
> 
> [at-loader] ./node_modules/aurelia-form/dist/aurelia-form.d.ts:227:8
>     TS1005: ';' expected.
> 
> [at-loader] ./node_modules/aurelia-form/dist/aurelia-form.d.ts:228:1
>     TS1128: Declaration or statement expected.
> 
> [at-loader] ./node_modules/aurelia-form/dist/aurelia-form.d.ts:228:16
>     TS1005: ';' expected.
> 
> [at-loader] ./node_modules/aurelia-form/dist/aurelia-form.d.ts:230:8
>     TS1005: ';' expected.
> 
> [at-loader] ./node_modules/aurelia-form/dist/aurelia-form.d.ts:231:1
>     TS1128: Declaration or statement expected.
> 
> [at-loader] ./node_modules/aurelia-form/dist/aurelia-form.d.ts:231:16
>     TS1005: ';' expected.
> 
> [at-loader] ./node_modules/aurelia-form/dist/aurelia-form.d.ts:233:8
>     TS1005: ';' expected.
> 
> [at-loader] ./node_modules/aurelia-form/dist/aurelia-form.d.ts:234:1
>     TS1128: Declaration or statement expected.
> 
> [at-loader] ./node_modules/aurelia-form/dist/aurelia-form.d.ts:234:16
>     TS1005: ';' expected.
> 
> [at-loader] ./node_modules/aurelia-form/dist/aurelia-form.d.ts:236:8
>     TS1005: ';' expected.
> 
> [at-loader] ./node_modules/aurelia-form/dist/aurelia-form.d.ts:237:1
>     TS1128: Declaration or statement expected.
> 
> [at-loader] ./node_modules/aurelia-form/dist/aurelia-form.d.ts:237:16
>     TS1005: ';' expected.
> 
> [at-loader] ./node_modules/aurelia-form/dist/aurelia-form.d.ts:239:8
>     TS1005: ';' expected.
> 
> [at-loader] ./node_modules/aurelia-form/dist/aurelia-form.d.ts:240:1
>     TS1128: Declaration or statement expected.
> 
> [at-loader] ./node_modules/aurelia-form/dist/aurelia-form.d.ts:240:16
>     TS1005: ';' expected.
> 
> [at-loader] ./node_modules/aurelia-form/dist/aurelia-form.d.ts:242:8
>     TS1005: ';' expected.
> 
> [at-loader] ./node_modules/aurelia-form/dist/aurelia-form.d.ts:243:1
>     TS1128: Declaration or statement expected.
> 
> [at-loader] ./node_modules/aurelia-form/dist/aurelia-form.d.ts:243:16
>     TS1005: ';' expected.
> 
> [at-loader] ./node_modules/aurelia-form/dist/aurelia-form.d.ts:245:8
>     TS1005: ';' expected.
> 
> [at-loader] ./node_modules/aurelia-form/dist/aurelia-form.d.ts:246:1
>     TS1128: Declaration or statement expected.
> 
> [at-loader] ./node_modules/aurelia-form/dist/aurelia-form.d.ts:246:16
>     TS1005: ';' expected.
> 
> [at-loader] ./node_modules/aurelia-form/dist/aurelia-form.d.ts:248:8
>     TS1005: ';' expected.
> 
> [at-loader] ./node_modules/aurelia-config/dist/aurelia-config.d.ts:24:16
>     TS1030: 'export' modifier already seen.
> 
> [at-loader] ./node_modules/aurelia-config/dist/aurelia-config.d.ts:49:2
>     TS1036: Statements are not allowed in ambient contexts.
> 
> [at-loader] ./node_modules/aurelia-form/dist/aurelia-form.d.ts:6:9
>     TS2305: Module '"C:/Git/pappt/aurelia-form-demo-typescript-webpack/node_modules/aurelia-config/dist/aurelia-config"' has no exported member 'Configuration'.
> 
> [at-loader] ./node_modules/aurelia-form/dist/aurelia-form.d.ts:10:15
>     TS2307: Cannot find module 'aurelia-form/decorator/index'.
> 
> [at-loader] ./node_modules/aurelia-form/dist/aurelia-form.d.ts:216:8
>     TS2304: Cannot find name 'declare'.
> 
> [at-loader] ./node_modules/aurelia-form/dist/aurelia-form.d.ts:218:3
>     TS2304: Cannot find name 'from'.
> 
> [at-loader] ./node_modules/aurelia-form/dist/aurelia-form.d.ts:219:8
>     TS2304: Cannot find name 'declare'.
> 
> [at-loader] ./node_modules/aurelia-form/dist/aurelia-form.d.ts:221:3
>     TS2304: Cannot find name 'from'.
> 
> [at-loader] ./node_modules/aurelia-form/dist/aurelia-form.d.ts:222:8
>     TS2304: Cannot find name 'declare'.
> 
> [at-loader] ./node_modules/aurelia-form/dist/aurelia-form.d.ts:224:3
>     TS2304: Cannot find name 'from'.
> 
> [at-loader] ./node_modules/aurelia-form/dist/aurelia-form.d.ts:225:8
>     TS2304: Cannot find name 'declare'.
> 
> [at-loader] ./node_modules/aurelia-form/dist/aurelia-form.d.ts:227:3
>     TS2304: Cannot find name 'from'.
> 
> [at-loader] ./node_modules/aurelia-form/dist/aurelia-form.d.ts:228:8
>     TS2304: Cannot find name 'declare'.
> 
> [at-loader] ./node_modules/aurelia-form/dist/aurelia-form.d.ts:230:3
>     TS2304: Cannot find name 'from'.
> 
> [at-loader] ./node_modules/aurelia-form/dist/aurelia-form.d.ts:231:8
>     TS2304: Cannot find name 'declare'.
> 
> [at-loader] ./node_modules/aurelia-form/dist/aurelia-form.d.ts:233:3
>     TS2304: Cannot find name 'from'.
> 
> [at-loader] ./node_modules/aurelia-form/dist/aurelia-form.d.ts:234:8
>     TS2304: Cannot find name 'declare'.
> 
> [at-loader] ./node_modules/aurelia-form/dist/aurelia-form.d.ts:236:3
>     TS2304: Cannot find name 'from'.
> 
> [at-loader] ./node_modules/aurelia-form/dist/aurelia-form.d.ts:237:8
>     TS2304: Cannot find name 'declare'.
> 
> [at-loader] ./node_modules/aurelia-form/dist/aurelia-form.d.ts:239:3
>     TS2304: Cannot find name 'from'.
> 
> [at-loader] ./node_modules/aurelia-form/dist/aurelia-form.d.ts:240:8
>     TS2304: Cannot find name 'declare'.
> 
> [at-loader] ./node_modules/aurelia-form/dist/aurelia-form.d.ts:242:3
>     TS2304: Cannot find name 'from'.
> 
> [at-loader] ./node_modules/aurelia-form/dist/aurelia-form.d.ts:243:8
>     TS2304: Cannot find name 'declare'.
> 
> [at-loader] ./node_modules/aurelia-form/dist/aurelia-form.d.ts:245:3
>     TS2304: Cannot find name 'from'.
> 
> [at-loader] ./node_modules/aurelia-form/dist/aurelia-form.d.ts:246:8
>     TS2304: Cannot find name 'declare'.
> 
> [at-loader] ./node_modules/aurelia-form/dist/aurelia-form.d.ts:248:3
>     TS2304: Cannot find name 'from'.
> 
> [at-loader] ./node_modules/aurelia-view-manager/dist/aurelia-view-manager.d.ts:87:31
>     TS2304: Cannot find name 'ViewEngine'.
> 
> [at-loader] ./node_modules/aurelia-view-manager/dist/aurelia-view-manager.d.ts:87:63
>     TS2304: Cannot find name 'ViewCompileInstruction'.
> 
> [at-loader] ./node_modules/aurelia-view-manager/dist/aurelia-view-manager.d.ts:87:101
>     TS2304: Cannot find name 'ResourceLoadContext'.
> 
> [at-loader] ./node_modules/aurelia-view-manager/dist/aurelia-view-manager.d.ts:87:131
>     TS2304: Cannot find name 'ViewFactory'.

--- a/spoonx.js
+++ b/spoonx.js
@@ -38,7 +38,7 @@ module.exports = {
    */
 
     /* concat js files
-     * concat: true
    */
+     concat: true
   }
 };

--- a/src/configuration.js
+++ b/src/configuration.js
@@ -3,7 +3,10 @@ import {Config} from './config';
 
 /**
  * Configuration class. A resolver for config namespaces which allows injection of the corresponding config segement into a class
- */
+ * @export
+ * @class Configuration
+*/
+
 @resolver()
 export class Configuration {
   /**

--- a/src/plugin-manager.js
+++ b/src/plugin-manager.js
@@ -8,7 +8,7 @@ import {FrameworkConfiguration} from 'aurelia-framework';
  * @export
  * @interface PluginDefinition
  */
-export interface PluginDefinition {
+interface PluginDefinition {
   /**
    * The moduleId as used by aurelia
    *

--- a/test/configuration.spec.js
+++ b/test/configuration.spec.js
@@ -1,5 +1,5 @@
 import {Config} from 'src/config';
-import {Configuration} from 'src/resolvers';
+import {Configuration} from 'src/configuration';
 import {Container} from 'aurelia-dependency-injection';
 import {InjectObjectTest} from 'test/resources/inject-test';
 

--- a/test/resources/inject-test.js
+++ b/test/resources/inject-test.js
@@ -1,5 +1,5 @@
 import {inject} from 'aurelia-dependency-injection';
-import {Configuration} from 'src/resolvers';
+import {Configuration} from 'src/configuration';
 
 @inject(Configuration.of('foo'))
 export class InjectObjectTest {

